webapp/portlet/src/main/webapp/vue-apps/general-settings/components/branding/SiteBranding.vue

```diff
@@ -272,7 +305,10 @@ export default {
         .then(() => this.$emit('saved'))
         .then(() => this.$root.$emit('alert-message', this.$t('generalSettings.savedSuccessfully'), 'success'))
         .catch(e => this.errorMessage = String(e))
-        .finally(() => this.$root.loading = false);
+        .finally(() => {
+          this.$root.$emit('refresh-iframe');
+          this.$root.loading = false;
+        });
```

Comment:
the refresh should be made only when successfully updated. Please copy `this.$root.$emit('refresh-iframe');` into `then(() => ...)` near message display
```suggestion
        .finally(() => this.$root.loading = false);
```

Reply:
done